dissect/target/filesystem.py

```diff
@@ -507,7 +533,7 @@ def walk(
         self,
         topdown: bool = True,
         onerror: Optional[Callable] = None,
-        followlinks: bool = False,
```

Comment:
I'm not too sure on renaming this argument, since `os.walk` still uses `followlinks` here. This would mean deviating from the `os.walk` convention.

Reply:
Ah, I wasn't aware that this was due to os.walk. It hasn't been changed in 3.11 or 3.12, so I'll remove the commit that changes this.